proposals/lip-0053.md

```diff
@@ -128,6 +128,7 @@ def verifyLivenessConditionForRegisteredChains(ccu: Transaction) -> None:
 def verifyCertificate(ccu: Transaction) -> None:
     certificateBytes = ccu.params.certificate      
```

Comment:
here, we also still assume that `params` is of type object. If this PR intends to change the definition of the type _Transaction_ globally, then it must also be changed here.

There seem to be some more occurrences in this file.

Reply:
I only applied all the changes to the Token module so far, not to Interoperability yet. Once we are aligned, I would also add the decoding step here (or we assume in this LIP that the type is actually CCU and not Transaction).